app/assets/javascripts/statistics.es6

```diff
@@ -0,0 +1,43 @@
+export class Statistics { 
+  constructor(collection_data, work_data) {
```

Comment:
Statistics is for how the tabs interact with the graph. It is scoped to the whole tab container not just works or collections.